test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -2290,6 +2291,9 @@ public async Task TestPackageManager_DowngradePackageFor_BottomtProject_Success(
         [Fact]
         public async Task TestPackageManager_CancellationTokenPassed()
         {
+            // Get exception messages in English
+            Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
Would it make sense to set this globally for all of our tests so that in the future we don't cause this to happen again?

Reply:
I think it would. The question is how to do it assembly/codebase wise?
Or do we want at least to set this on a test class level instead of individual tests level?